cdap-standalone/bin/cdap.bat

```diff
@@ -205,6 +211,7 @@ SET "CLASSPATH=%CDAP_HOME%\lib\*;%CDAP_HOME%\conf\;"
 CALL :CHECK_WINDOWS
 CALL :CHECK_JAVA
 CALL :CREATE_LOG_DIR

```

Comment:
Should we not have a `IF %ERRORLEVEL% NEQ 0 GOTO FINALLY` before line 213, just as we do in the `:SDK_START` function?